low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -539,6 +541,56 @@ def process(self):
         logger.info(f"{self.name} interface not implemented.")
 
 
+class PreUpgradeCmd(Cmd):
+    """PreUpgrade support for SSPL componenet."""
+
+    name = "pre_upgrade"
+
+    def __init__(self, args):
+        super().__init__(args)
+
+    def validate(self):
+        # Common validator classes to check Cortx/system wide validator
+        pass
+
+    def process(self):
+        logger.info(f"{self.name} interface not implemented.")
+
+
+class PostUpgradeCmd(Cmd):
+    """PostUpgrade support for SSPL componenet."""
+
+    name = "post_upgrade"
+
+    def __init__(self, args):
+        super().__init__(args)
+
+    def validate(self):
+        # Common validator classes to check Cortx/system wide validator
+        pass
+
+    def process(self):
+        new_conf_url = 'yaml:///opt/seagate/cortx/sspl/conf/sspl.conf.LR2.yaml'
+        existing_conf_url = 'yaml:///etc/sspl.conf'
+        merged_conf_url = 'yaml:///opt/seagate/cortx/sspl/tmp/merged.conf'
+        # Only proceed if both existing and new config path are present
+        for filepath in [existing_conf_url, new_conf_url]:
+            if not os.path.exists(filepath.split(":/")[1]):
+                logger.info("Exiting, File %s does not exists" % filepath)
```

Comment:
How this print is helping? Need to be clear that no config upgrade involved as existing or new config is not present.
Also how can these files  be missing in any case ?
I think it can be silently ignored if no config upgrade is seen needed or keep logs at debug level

Reply:
`Also how can these files be missing in any case ?` Old file may be missing, if someone has removed it manually after removing rpm.
Updated log